pkg/systemd/reporting.jsx

```diff
@@ -312,7 +312,7 @@ const BusWorkflowRow = ({ problem, client, workflow }) => {
                         /* GLib encodes errors for transport over the wire,
                          * but we don’t have a good way of decoding them without calling into GIO.
                          *
-                         * https://developer-old.gnome.org/gio/stable/gio-GDBusError.html#g-dbus-error-encode-gerror
+                         * https://dbus.freedesktop.org/doc/dbus-glib/dbus-glib-DBusGError.html#enum%20DBusGError
```

Comment:
This is not the same thing.   One of those is from libdbus and the other is from gio (glib).   https://docs.gtk.org/gio/type_func.DBusError.encode_gerror.html is the first google hit for what I think you're looking for.